lib/u2f/register_response.rb

```diff
@@ -20,6 +21,10 @@ def self.load_from_json(json)
         raise RegistrationError, code: data['errorCode']
       end
 
+      if !data.key?('clientData') || !data.key?('registrationData')
+        raise RegistrationError, code: BAD_REQUEST
```

Comment:
These error codes normally come from the browser. It might make more sense to pass a `message:` argument instead of a `code:`.